indico/modules/events/abstracts/templates/reviewing/public.html

```diff
@@ -15,27 +15,44 @@ <h3 class="f-self-stretch">
+                                <br>
+                                {%- trans -%}
+                                    Withdrawing this abstract will permanently delete its corresponding contribution.
+                                {%- endtrans -%}
+                            {% elif not can_manage %}
+                                <br>
+                                {% trans %}This operation is irreversible.{% endtrans %}
+                            {% endif %}
+                        {% endset %}
                         <a title="{% trans %}Withdraw abstract{% endtrans %}"
                            class="i-button text-color color-on-hover danger"
                            data-href="{{ url_for('.withdraw_abstract', abstract) }}"
                            data-method="POST"
                            data-update="#reviewing-page"
                            data-replace-update
                            data-title="{% trans %}Withdraw Abstract{% endtrans %}"
-                           data-confirm="{% trans %}Do you really want to withdraw this abstract? This operation is irreversible.{% endtrans %}">
+                           data-confirm="{{ message }}">
                             {% trans %}Withdraw{% endtrans %}
                         </a>
-                    {% elif abstract.state.name == 'withdrawn' and can_manage %}
-                        <a title="{% trans %}This abstract has been withdrawn but can be restored using this option{% endtrans %}"
-                           data-href="{{ url_for('.reset_abstract_state', abstract) }}"
-                           data-method="POST"
-                           data-update="#reviewing-page"
-                           data-replace-update
-                           data-title="{% trans %}Restore Abstract{% endtrans %}"
-                           data-confirm="{% trans %}Do you really want to restore this abstract? This operation is irreversible.{% endtrans %}"
-                           class="i-button accept icon-lamp">
-                            {% trans %}Restore{% endtrans %}
+                    {% elif abstract.state.name == 'accepted' %}
+                    <a title="{% trans %}This abstract has been withdrawn but can be restored using this option{% endtrans %}"
+                       class="i-button accept icon-lamp"
+                       data-href="{{ url_for('.reset_abstract_state', abstract) }}"
+                       data-method="POST"
+                       data-update="#reviewing-page"
+                       data-replace-update
+                       data-title="{% trans %}Restore Abstract{% endtrans %}"
+                       data-confirm="{% trans %}Do you really want to restore this abstract?{% endtrans %}">
+                        {% trans %}Restore{% endtrans %}
+                    </a>
+                    <a title="{% trans %}Withdraw abstract{% endtrans %}"
```

Comment:
Guess what you forgot here ;)

Depending on how long the message will be also use a `{% set %}` above instead of inlining the message